clusters/development/flux-system/gotk-sync.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
 spec:
   interval: 1m0s
   ref:
-    branch: master
+    branch: upgrade-flux-api
```

Comment:
Should this be "master"?

Reply:
the `flux-system` directories are managed by flux so this will be overwritten when bootstrapping flux in the cluster.